### PR TITLE
Test chinese version branch

### PR DESCRIPTION
**Pull request by katcom**
Translation some of values into Chinese.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@
 /android-refimpl-app.externalNativeBuild
 android-refimpl-app/app/nativelibs/armeabi/libjni-c-toxcore.so
 android-refimpl-app/app/nativelibs/armeabi-v7a/libjni-c-toxcore.so
+
```
Comment (zoff99) [inline .gitignore]: remove unused newline

**Review by zoff99 — CHANGES_REQUESTED** [review]

**Comment by zoff99** [comment]
@iphydf if you have time can you check those texts please?

**Comment by linsui** [comment]
@zoff99 Could you tell me what I can do to help translate this app?

**Comment by zoff99** [comment]
there is a new PR for this. this one failed to compile

#99

**Pull request closed without merging**